### PR TITLE
Add CAN-FD support for Renesas RA6, RA4

### PR DESCRIPTION
**Pull request by khoa-nguyen-18**
Add CAN-FD support on these boards:
- ek_ra6e2
- ek_ra4e2

### CODE REVIEW

File: boards/renesas/ek_ra4e2/ek_ra4e2.dts [hunk 1]
```diff
@@ -76,3 +84,24 @@
 &ioport2 {
 	status = "okay";
 };
+
```
Comment (soburi) [inline boards/renesas/ek_ra4e2/ek_ra4e2.dts:87]: This part is not related to CAN. Please split to another PR.

Comment (soburi) [reply boards/renesas/ek_ra4e2/ek_ra4e2.dts:87]: OK. Please mention that in your commit message.

**Review by soburi — DISMISSED** [review]

**Comment by khoa-nguyen-18** [comment]
- Rebase main to solve the conflicts
- Remove can-fd support for fpb_ra6e2

**Review by soburi — APPROVED** [review]

**Review by thaoluonguw — APPROVED** [review]
